rdflib/plugins/serializers/hext.py

```diff
@@ -101,14 +104,14 @@ def _hex_line(self, triple, context):
             else:
                 language = ""
 
-            return '["%s", "%s", "%s", "%s", "%s", "%s"]\n' % (
+            return json.dumps([
```

Comment:
I just noticed this today actually when debugging some other tests and was about to add some xfails, glad it is fixed :+1: